Description: Remove simplejson usage
 It serves no purpose and it fails the JSON tests included
 in Raven when present.
 .
 raven (5.18.0-1) UNRELEASED; urgency=medium
 .
   * Initial release.
Author: Gilles Dubuc <gilles@wikimedia.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- raven-5.18.0.orig/raven/utils/json.py
+++ raven-5.18.0/raven/utils/json.py
@@ -11,10 +11,7 @@ from __future__ import absolute_import
 import codecs
 import datetime
 import uuid
-try:
-    import simplejson as json
-except ImportError:
-    import json
+import json
 
 try:
     JSONDecodeError = json.JSONDecodeError
